pkgs/applications/office/qownnotes/default.nix

```diff
@@ -14,7 +15,7 @@ mkDerivation rec {
     sha256 = "2c86d66ae427bdcd16d706b982cedaa669a27340f7819fc97a8e2b24c709e74f";
   };
 
-  nativeBuildInputs = [ qmake qttools ];
+  nativeBuildInputs = [ qmake qttools wrapGAppsHook ];
```

Comment:
Sorry for the late response. No, I'm not running GNOME, but bspwm without a DE.
Do you think the crashes I'm experiencing are just because I'm just missing some library on my system?
Other applications (Qt and GTK) can launch file dialogs just fine...